packages/core/src/modules/credentials/formats/indy/IndyCredentialFormatService.ts

```diff
@@ -211,13 +212,19 @@ export class IndyCredentialFormatService extends CredentialFormatService<IndyCre
   public async processOffer({ attachment, credentialRecord }: FormatProcessOptions) {
     this.logger.debug(`Processing indy credential offer for credential record ${credentialRecord.id}`)
 
-    const credOffer = attachment.getDataAsJson<Indy.CredOffer>()
+    const credentialOffer = attachment.getDataAsJson<Indy.CredOffer>()
 
-    if (!credOffer.schema_id || !credOffer.cred_def_id) {
+    if (!credentialOffer.schema_id || !credentialOffer.cred_def_id) {
       throw new CredentialProblemReportError('Invalid credential offer', {
         problemCode: CredentialProblemReportReason.IssuanceAbandoned,
       })
     }
+
+    credentialRecord.metadata.set(CredentialMetadataKeys.IndyCredential, {
+      credentialDefinitionId: credentialOffer.cred_def_id,
+      schemaId: credentialOffer.schema_id,
+    })
```

Comment:
This was removed recently as we discussed that the record will only contain the values we have accept. When we process an offer we haven't accepted the offer yet.

This makes it so that I can always check for the values I last accepted in the metadata, as otherwise it can be either the data I received, or the data I sent.

If we want to revert this (which I'd like to have a more in depth discussion about) it'll be a breaking change.